client/src/app/account-card/account-card.component.scss

```diff
@@ -0,0 +1,49 @@
+.account-container {
+  margin-left: 5rem;
+  margin-top: 2rem;
+  position: relative;
+  width: max-content;
+}
+
+.account-card {
+  width: 18rem;
+  height: 8rem;
+  background-image: linear-gradient(
+    to right bottom,
+    #f4497d,
```

Comment:
Create, please, file with color constants somewhere in the root of the project in folder shared. Add all colors in this file. When you need color, you should import file with constants and then you can use it. With this approach you can reuse constants